cmd/brimstone/main.go

```diff
@@ -20,87 +20,79 @@ import (
 	"github.com/labstack/echo/v4/middleware"
 
 	"gorm.io/driver/postgres"
+	"gorm.io/driver/sqlite"
 	"gorm.io/gorm"
+
+	"github.com/caarlos0/env/v6"
```

Comment:
Is this the latest version?  Looks like v10 might be the latest version.  Recommend using the latest version.